### PR TITLE
Release/2.6.0-RC.1

### PR DESCRIPTION
**Pull request by henrypalacios**
# Release v2.6.0-RC.1

:warning: DO NOT SQUASH :warning:

## Main features:
- Added transactions view
- Changes on header and footer (network aware)
- Misc bug fixes and improvements

### Features
cd395a69 header links (#953)
e7064099 transaction view (#882)
b082c028 Tx table storybook (#897)
9678a7b1 Update XDAI, WXDAI and GNO token logos (#986)
57b33bd9 Adding queue_rules (#955)
d97517e9 Merge remote-tracking branch 'origin/master' into develop
b734984c Search tx every network (#933)
2f911bef Make contracts urls on footer network aware by receiving networkId changes (#934)
6f227a8a Search for tx on search bar (#899)
51885adb Merge pull request #926 from gnosis/release/2.5.0_update_develop
7f147782 Merge branch 'develop' into release/2.5.0_update_develop
fb1d7f2f 917 rename xdai to gc (#918)
ee60fd4f Wire up TransactionDetails page with Tx API (#908)
6795db3c Adding getTxOrders to connect Tx API (#900)

### Dependency Updates
a43a745f Bump follow-redirects from 1.14.5 to 1.14.7 (#987)
c9cef1f6 Bump playwright from 1.16.3 to 1.17.2 (#939)
8411ebd3 Bump @types/styled-components from 5.1.15 to 5.1.19 (#928)
6f463ede Bump @types/node from 14.17.34 to 14.18.1 (#916)

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
Looking great, Henry.

One small thing:

In the tx details we are looking at a list of `orders` instead of `transactions`
Screen Shot 2022-01-31 at 18 57 37

That's all, otherwise looks good to me.

Would also like to have @elena-zh to validate it as well

File: src/components/transaction/TransactionTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+            </EmptyItemWrapper>
+          </td>
+        </tr>
+      )
+    } else {
+      tableContent = (
+        <>
+          {items.map((item, i) => (
+            <RowTransaction
+              key={`${item.shortId}-${i}`}
```
Comment (anxolin) [inline src/components/transaction/TransactionTable/index.tsx:259]: Curious why the `${i}` was needed here. Can't we just identify the order by its ID alone? Wouldn't this be better?

**Review by anxolin — APPROVED** [review]
This is a great version! Congrats.


I would only suggest to change the colors of Gnosis Chain:

Current one has no contrast:
image


My proposal: 
```
    background: rgba(7,121,91,1.00);
    color: white;
}
```
image


cc/ @biocom

**Comment by elena-zh** [comment]
> Looking great, Henry.
> 
> One small thing:
> 
> In the tx details we are looking at a list of `orders` instead of `transactions` Screen Shot 2022-01-31 at 18 57 37
> 
> That's all, otherwise looks good to me.
> 
> Would also like to have @elena-zh to validate it as well

I created a separate issue for this #1014

**Comment by anxolin** [comment]
Why do we open orders in a new tab? I think it should just navigate

**Comment by elena-zh** [comment]
> This is a great version! Congrats.
> 
> I would only suggest to change the colors of Gnosis Chain:
> 
> Current one has no contrast: image
> 
> My proposal:
> 
> ```
>     background: rgba(7,121,91,1.00);
>     color: white;
> }
> ```
> 
> image
> 
> cc/ @biocom

I have created #1015 task for this

**Comment by elena-zh** [comment]
> Why do we open orders in a new tab? I think it should just navigate

@anxolin , I think, we used the same logic as we do for the User details page: Orders are opened in a new tab there. 
See the video: https://watch.screencastify.com/v/5DPR1ZKKwCSTcCCqTIKG

I think, we should follow the same logic everywhere. So if we want to open orders in the same tab from each place in the app, I will create an enhancement for this. 

Thanks

**Review by elena-zh — CHANGES_REQUESTED** [review]
I'd like to request changes due to this issue: #1017 
Besides, it would be nice to take a look issues mentioned and reported above (#1016, #1014)

Issue #1018 is not exactly related to the current PR, but it would be nice to fix before the release.

**Comment by anxolin** [comment]
I'd love to make another request. Could we also add a link to Analytics in the header with https://dune.xyz/gnosis.protocol/Gnosis-Protocol-V2 . It could go after community

**Comment by anxolin** [comment]
> @anxolin , I think, we used the same logic as we do for the User details page: orders are opened in a new tab from there.
See the video: https://watch.screencastify.com/v/5DPR1ZKKwCSTcCCqTIKG
> 
> I think, we should follow the same logic everywhere. So if we want to open orders in the same tab from each place in the app, I will create an enhancement for this.

Thanks for bringing this app. I think we should open all those links in the same tab, just a normal navigation. Reason: it happens within the same app, and an user can always right click and open in a new tab if they want

**Comment by elena-zh** [comment]
> I'd love to add another request. Could we also add a link to Analytics in the header with https://dune.xyz/gnosis.protocol/Gnosis-Protocol-V2 . It could go after community

Added a task #1019

**Comment by elena-zh** [comment]
> > @anxolin , I think, we used the same logic as we do for the User details page: orders are opened in a new tab from there.
> > See the video: https://watch.screencastify.com/v/5DPR1ZKKwCSTcCCqTIKG
> > I think, we should follow the same logic everywhere. So if we want to open orders in the same tab from each place in the app, I will create an enhancement for this.
> 
> Thanks for bringing this up. I think we should open all those links in the same tab, just a normal navigation. Reason: it happens within the same app, and an user can always right click and open in a new tab if they want

Created #1020 issue for this.

**Comment by anxolin** [comment]
One other small suggestion. Not for this release, but probably nice for other ones.

- I really like how now, in the details you click in the TX or ADDRESS and takes you to the relevant page in the explorer
- In both cases, we used to send the user to Etherscan, wich is appreciated by them, since its something very familiar
- Now we still go to etherscan, by clicking on the address in the top of both pages
- HOWEVER: it's not clear enough that this will be the case. I feel the tx could have a label saying (`View in Etherscan`) to be more explicit. Otherwise you never know which links take you out of the explorer

@biocom @alfetopito @elena-zh WDYT?

**Comment by elena-zh** [comment]
@anxolin , great idea!
'View on Etherscan/Blockscount' will definitely explain a user where he/she is going to be navigated when pressing on the link address/TxHash at the top of the page. 

But then, we will need to add the  message 'View on Etherscan/Blockscount' near the link everywhere in the app for consistency
image
image
image
image

WDYT?

**Comment by anxolin** [comment]
> But then, we will need to add the message 'View on Etherscan/Blockscount' near the link everywhere in the app for consistency

Good point! although  i think for tokens is too much, at least for now :) 
Maybe if in the future the token doesn't take you to Etherscan we can follow the same logic. I could see a page for the token, showing some stats on top (total traded volume, 24h traded volume), also latest trades. So if we had this, we would be coherent just by adding a link in the top taking you to etherscan.

For now, we can make an exception for tokens imo

**Comment by elena-zh** [comment]
@anxolin , I have created an enhancement for this #1021

**Comment by anxolin** [comment]
I will merge this, so we can have a staging URL with this version. I think its nice, because allow us to visualize transactions of settlements in production (the PR link works with development only).

I think the relevant improvements suggested in this PR can be done as independent PRs against `release/2.6.0`

**Comment by elena-zh** [comment]
> I will merge this, so we can have a staging URL with this version. I think its nice, because allow us to visualize transactions of settlements in production (the PR link works with development only).
> 
> I think the relevant improvements suggested in this PR can be done as independent PRs against `release/2.6.0`

@anxolin , we have related issue unfixed #1017 . 
And from my perspective it is important to fix it before merging to master..
Could you please revert your merge?

**Comment by alfetopito** [comment]
> 

What about adding the Etherscan icon like we do on CowSwap
Screen Shot 2022-02-01 at 09 54 56

But not to every single link, just where it's not obvious.
I wouldn't for example add it to token links

**Pull request merged**